subprojects/core/src/main/java/org/gradle/internal/buildevents/BuildExceptionReporter.java

```diff
@@ -64,6 +68,7 @@
  */
 public class BuildExceptionReporter implements Action<Throwable> {
     private static final String NO_ERROR_MESSAGE_INDICATOR = "(no error message)";
+    public static final String SKIPPABLE_ERROR_MESSAGE_SUFFIX = "due to earlier error below:";
```

Comment:
❓ I don't understand why this constant is here. It's not used here, and the only usage is in `ErrorHandlingModuleComponentRepository`, if you don't count tests.

Reply:
Left over from an intermediate stage - I'll clean this up.